docs/workflows/genomic_characterization/freyja.md

```diff
@@ -201,7 +215,7 @@ This workflow runs on the sample level.
 
 </div>
 
-### Freyja_FASTQ Analysis Tasks
+#### Analysis Tasks
 
 ??? task "`read_QC_trim_pe` Details"
 
```

Comment:
can you please remove the level 5 headings here? the toc is getting pretty long and we may be better served w/o them in the toc

Reply:
Done!!